chapters/data-collection.tex

```diff
@@ -1,362 +1,299 @@
+
+There is debate over how individual questions should be identified: formats like \texttt{hq\_1} are hard to remember and unpleasant to reorder, but formats like \texttt{hq\_asked\_about\_loans} quickly become cumbersome.
+We recommend using descriptive names with clear prefixes so that variables 
+within a module stay together when sorted 
+alphabetically.\sidenote{\url{https://medium.com/@janschenk/variable-names-in-survey-research-a18429d2d4d8}}
+ Variable names should never include spaces or mixed cases (all lower case is 
+best). Take care with the length: very long names will be cut off in certain 
+software, which could result in a loss of uniqueness. We discourage explicit 
+question numbering, as it discourages re-ordering, which is a common 
+recommended change after the pilot. In the case of follow-up surveys, numbering 
+can quickly become convoluted, too often resulting in variables names like 
+'ag\_15a', 'ag\_15\_new', 'ag\_15\_fup2', etc.
+
+Questionnaires must include ways to document the reasons for \textbf{attrition}, treatment \textbf{contamination}, and \textbf{loss to follow-up}.
+\index{attrition}\index{contamination}
+These are essential data components for completing CONSORT records, a standardized system for reporting enrollment, intervention allocation, follow-up, and data analysis through the phases of a randomized trial of two groups.
+\sidenote[][-3.5cm]{Begg, C., Cho, M., Eastwood, S., Horton, R., Moher, D., Olkin, I., Pitkin,	R., Rennie, D., Schulz, K. F., Simel, D., et al. (1996). Improving the quality of	reporting of randomized controlled	trials: The CONSORT statement. \textit{JAMA},	276(8):637--639}
+
+
+
+\subsection{Content-focused Pilot}
+A \textbf{survey pilot}\sidenote{\url{https://dimewiki.worldbank.org/wiki/Survey_Pilot}} is the final step of questionnaire design.
+A content-focused pilot\sidenote{\url{https://dimewiki.worldbank.org/wiki/Piloting_Survey_Content}}  is best done on pen-and-paper, before the questionnaire is programmed.
+The objective is to improve the structure and length of the questionnaire, refine the phrasing and translation of specific questions, and confirm coded response options are exhaustive.\sidenote{\url{https://dimewiki.worldbank.org/index.php?title=Checklist:_Refine_the_Questionnaire_(Content)&printable=yes}}
+In addition, it is an opportunity to test and refine all survey protocols, such as how units will be sampled or pre-selected units identified. The pilot must be done out-of-sample, but in a context as similar as possible to the study sample.
+
```

Comment:
One-line paragraphs seem a bit weird to me. I suggest removing the new line. But this small, so happy to go with it if I'm the only one who's bothered by it.